ztp/gitops-subscriptions/argocd/resource-hook-example/policygentemplates/common-ranGen.yaml

```diff
@@ -0,0 +1,55 @@
+---
```

Comment:
This is a duplicate of ztp/ztp-policy-generator/ranPolicyGenTempExamples/common-ranGen.yaml. We should have only one reference implementation and pull from there.